README.md

```diff
@@ -213,8 +213,9 @@ print("Recall for two batches:", np.mean(labels.reshape(-1) == np.arange(len(dat
 You can install from sources:
 ```bash
 apt-get install -y python-setuptools python-pip
-pip3 install pybind11 numpy setuptools
-python3 setup.py install
+git clone git@github.com:nmslib/hnswlib.git
```

Comment:
Looks like that it should be changed to the `git clone https://github.com/nmslib/hnswlib.git`

Reply:
Agree. The `https://` scheme is probably more user friendly for github users who haven't setup ssh keys and those that have will already know how to clone the source. 👍